config/externalnamenottested.go

```diff
@@ -288,13 +297,28 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 
 	// artifactregistry
 	//
+	// Imported by using the following format: projects/{{project}}/locations/{{location}}/repositories/{{repository_id}}
+	"google_artifact_registry_repository": config.TemplatedStringAsIdentifier("repository_id", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.location }}/repositories/{{ .external_name }}"),
 	// Imported by using the following format: projects/{{project}}/locations/{{location}}/repositories/{{repository}} roles/artifactregistry.reader user:jane@example.com
 	"google_artifact_registry_repository_iam_member": config.TemplatedStringAsIdentifier("repository", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.location }}/repositories/{{ .external_name }} {{ .parameters.role }} {{ .parameters.member }}"),
 
+	// beyondcorp
+	//
+	// Imported by using the following projects/{{project}}/locations/{{region}}/appConnections/{{name}}
+	"google_beyondcorp_app_connection": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.region }}/appConnections/{{ .external_name }}"),
+	// Imported by using the following projects/{{project}}/locations/{{region}}/appConnectors/{{name}}
+	"google_beyondcorp_app_connector": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.region }}/appConnectors/{{ .external_name }}"),
+	// Imported by using the following projects/{{project}}/locations/{{region}}/appGateways/{{name}}
+	"google_beyondcorp_app_gateway": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.region }}/appGateways/{{ .external_name }}"),
```

Comment:
We may consider marking the `region` field as required.

Reply:
Already marked in config.go